src/libraries/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -18,16 +18,10 @@ public enum Architecture
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public static System.Runtime.InteropServices.OSPlatform Android { get { throw null; } }
         public static System.Runtime.InteropServices.OSPlatform Browser { get { throw null; } }
```

Comment:
I think `Browser` should be removed as well, since we added it in `net5.0`

Reply:
@adamsitnik mentioned he'll send another PR for removing `Browser` once #40457 gets merged.

Reply:
@eerhardt I would prefer to do it in the next PR where I will replace all usages of `RuntimeInfromation.IsOSPlatform(XYZ)` with  `OperatingSystem.IsXYZ` new method calls

Reply:
Ok - I'm not sure I understand the difference between `Browser` and `iOS` in this area, but that is fine with me.

---
In reply to: [466574218](https://github.com/dotnet/runtime/pull/40373#discussion_r466574218) [](ancestors = 466574218)

Reply:
`Browser` was added earlier in 5.0 and has consumers; `iOS` and these others were just added in Preview 8 and doesn't yet have consumers.

Reply:
ok, I've removed browser and replace it's usages:

```diff
-RuntimeInformation.IsOSPlatform(OSPlatform.Browser)
+OperatingSystem.IsBrowser()
```

since #40457 has not been merged yet, the CI is going to go red.

Once #40457 CI is green and merged I am going to sync this PR, wait until it goes green and merge it. And then send a PR to ASP.NET repo